tests/codebuild/create_tests.sh

```diff
@@ -67,6 +67,27 @@ function verify_integration_tests
   verify_test HyperparameterTuningJob spot-xgboost-mnist-hpo 20m Completed
   verify_test HyperparameterTuningJob xgboost-mnist-hpo-custom-endpoint 20m Completed
   verify_test TrainingJob xgboost-mnist-debugger 20m Completed
+  # Verify that debug job has status
+  verify_debug_test TrainingJob xgboost-mnist-debugger NoIssuesFound
+}
+
+# This function verifies that a given debug job has specific status
+# Parameter:
+#    $1: CRD type
+#    $2: Instance of CRD
+#    $3: Single debug job status
+function verify_debug_test
+{
+  local crd_type="$1"
+  local crd_instance="$2"
+  local debug_job_status="$3"
+  # TODO extend this for multiple debug job with debug job statuses parameter
+  if [ "$(kubectl describe "$crd_type" "$crd_instance" | grep "Rule Evaluation Status" | grep "$debug_job_status" | wc -l)" -gt "0" ]; then
```

Comment:
Since we already have `jq` installed in this image, we can leverage the json output type for kubectl. Please check that this command actually works, since I haven't actually verified it against a running debugger job.
```
if [ $(kubectl get "$crd_type" "$crd_instance" -o json | jq -r ".status.debugRuleEvaluationStatuses[0].ruleEvaluationStatus") == "$debug_job_status" ]; then
```

Reply:
We don't have the  debug job status in `get` output. I did not find an option to support column in `get` output conditionally.

Reply:
Running `get` with the `-o json` gives the same information as `describe` but in JSON format. Please try the above command and see if the jq output is the EvaluationStatus.